services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -54,7 +54,25 @@ export type ActionDetails = {
     lockMessage?: string;
     version?: number;
 };
-
+let totalActions = 0;
```

Comment:
you don't have to keep count of the number of actions.

you can get that from the actions store. 

https://github.com/freiheit-com/kuberpult/blob/e419e420d24400bcc385711a2814ff692555e46d/services/frontend-service/src/ui/utils/store.tsx#L44

Reply:
and in the `actionsCart` you can use that hook.
https://github.com/freiheit-com/kuberpult/blob/e419e420d24400bcc385711a2814ff692555e46d/services/frontend-service/src/legacy-ui/ActionsCart/ActionsCart.tsx#L91-L93

```
// in Actions Cart
const numberOfActions = useNumberOfActions();
```

```
// in the store.ts
useNumberOfActions = useActions().length;
```

Reply:
The actionsCart is in the legacy ui and is not present in the new ui. Perhaps you meant something else?

Reply:
I updated the method of keeping track of the actions in cart as you mentioned

Reply:
You're correct, I meant the `sidebar.tsx` file.

Reply:
Done